alibi_detect/base.py

```diff
@@ -144,8 +144,10 @@ def from_config(cls, config: dict):
         config
             A config dictionary matching the schema's in :class:`~alibi_detect.saving.schemas`.
         """
-        # Check for exisiting version_warning. meta is pop'd as don't want to pass as arg/kwarg
-        version_warning = config.pop('meta', {}).pop('version_warning', False)
+        # Check for existing version_warning. meta is pop'd as don't want to pass as arg/kwarg
+        meta = config.pop('meta', None)
+        meta = {} if meta is None else meta
```

Comment:
Why not `meta = config.pop('meta', {})`? Ease of reading?

Reply:
See previous comment.